_docs/flavour/templates.md

```diff
@@ -83,15 +83,15 @@ The main entry point to template engine is the
 [Templates](https://github.com/konsoletyper/teavm-flavour/blob/master/templates/src/main/java/org/teavm/flavour/templates/Templates.java) class.
 For basic usage you need only these methods:
 
-* `Templates.bind(Object, HTMLElement|String)` which binds an instance of view class to the element of static HTML,
-  specified either as an `HTMLElement` instance or by id.
-* `Templates.update()` forces updating of all bound templates.
+* `Templates.bind(Object, HTMLElement|String)` which binds an instance of view class to an element from a static HTML file,
+  specified either as an `HTMLElement` instance or by String id.
```

Comment:
1. Why 'String' here
2. Why 'String' starts with capital letter?

Reply:
Agreed, String doesn't add anything here. I will take it out.